apps/docs/src/app/core/component-docs/select/examples/select-nested-options/select-nested-options.component.html

```diff
@@ -1,4 +1,5 @@
 <fd-select [(value)]="selectedValue"
+           [extendedBodyTemplate]="true"
```

Comment:
`extendedBodyTemplate` is added to let Option Components know if the text should be wrapped in `fd-list-title` directive. It's mostly to keep overflow in correct way